osu.Game/Screens/GameModeWhiteBox.cs

```diff
@@ -113,14 +114,12 @@ public GameModeWhiteBox()
                         },
                     }
                 },
-                popButton = new Button
+                popButton = new BackButton
                 {
-                    Text = @"Back",
-                    RelativeSizeAxes = Axes.X,
-                    Size = new Vector2(0.1f, 40),
+                    RelativeSizeAxes = Axes.None,

```

Comment:
if you're gonna do this here, i'd just make the default not have a RelativeSizeAxes set.